src/clickfunnels/main.ts

```diff
@@ -82,6 +92,38 @@ document.addEventListener('DOMContentLoaded', () => {
     return originalFetch.apply(this, args);
   };
 
+  // Get the button element
+  const button = document.querySelector('.elButton');
+
+  // Add a click event listener to the button
+  button.addEventListener('click', () => {
+    // Get the text within the button
+    const buttonText = button.textContent.trim();
+
+    // Get the query params
+    const url = new URL(window.location.href);
+    const queryParams: Record<string, string> = {};
+    for (const [key, value] of url.searchParams.entries()) {
+      queryParams[_.camelCase(key)] = value;
+    }
+
+    // Additional data
+    const additionalData = {
+      url: url.toString(),
+      path: url.pathname,
+      userAgent: navigator.userAgent,
+      fbc: Cookies.get('fbc'),
+      fbp: Cookies.get('fbp'),
+    };
+
```

Comment:
The term `additionalData` seems a bit vague to me.

Reply:
Not going to change this because I think I'll be breaking this out into a plugin, thus removing the need for this variable name